### ISSUE TITLE
Filename of binary when using -u mode

### ISSUE THREAD

**Issue by guidoferreyra** [googlefonts/fontmake#701]
Hi I have a question, hope anyone can help me :)

When using fontmake with a single ufo and expecting to have one compiled font, the filename of the font is the same of the ufo. But when building from design space every instance gets its filename as expected, based on the family and the style name. 

Is not possible to have this with a single ufo?

Based on what I read on fontmake help, I assume this could be done using `--output-path` but I’m using fontmake inside shell script without knowing what I’m building and avoiding to hardcode any names or paths.

Thanks in advance

**Comment by alerque** [comment]
Using `--output-path` is the correct solution. You don't have to hard code a path, you can make it adaptive. Presumably you have an input filename, you can have your script transform the `.ufo` filename to the output path you expect and use that.

Note there is an issue with UFO instance output being uncontrollable (see #693) but that's not the same problem you're having.

**Comment by guidoferreyra** [comment]
Hi alerque, thanks for commenting!
I understand what you say but what I need to do is to build the name from the ufo Family name and style name. Get this info, to build the new filename, probably can done with ufoLib but I was wondering if fontmake doesn’t already have an option for this.

**Comment by alerque** [comment]
I don't think there is, at least I didn't find one when I was rigging [fontship](https://github.com/theleagueof/fontship) to do something similar. It has the data internally but I don't know of an interface to cleanly pull it or use it to reformat the output name yourself.

On the other hand if you name UFO with a sensible name such as `Family-Style.ufo` then you won't need to parse the UFO itself to get that info. Since you have a single UFO anyway this shouldn't be too hard to achieve.

**Comment by guidoferreyra** [comment]
> On the other hand if you name UFO with a sensible name such as Family-Style.ufo then you won't need to parse the UFO itself to get that info. Since you have a single UFO anyway this shouldn't be too hard to achieve.
I agree. Anyway I finally ended up getting the info for the name with a few lines using FontParts.

Thanks!
